### PR TITLE
Bump version of wheel in requirements.txt

### PR DESCRIPTION
**Pull request by dstansby**
Replaces https://github.com/matplotlib/cheatsheets/pull/137. I figured it was worth waiting on bumping other dependencies until we've worked out how to bump the Matplotlib dependency.

### CODE REVIEW

**Review by timhoffm — APPROVED** [review]

File: requirements/requirements.txt [hunk 1]
```diff
@@ -120,10 +119,8 @@ pyyaml==6.0
     # via pre-commit
 requests==2.27.1
     # via sphinx
-scipy==1.8.0
+scipy==1.11.4
```
Comment (QuLogic) [inline requirements/requirements.txt:122]: This seems unrelated to `wheel`?

Comment (dstansby) [reply requirements/requirements.txt]: Ah good catch, that's because I accidentally ran pip-compile on py311 instead of 310. Should be fixed now, with just wheel being updated.

**Review by QuLogic — APPROVED** [review]
